project/widgets.py

```diff
@@ -65,8 +67,10 @@ class Led(QtWidgets.QCheckBox):
     def __init__(self):
         QtWidgets.QCheckBox.__init__(self)
         self.setDisabled(True)
-        StyleSheet = 'QCheckBox::indicator:checked {image: url(C:/Users/John/Desktop/PyCMDS/project/widget files/checkbox_checked.png);}'
-        StyleSheet += 'QCheckBox::indicator:unchecked {image: url(C:/Users/John/Desktop/PyCMDS/project/widget files/checkbox_unchecked.png);}'
+        path = str(__here__).replace("\\", "/")
+        StyleSheet = f'QCheckBox::indicator:checked {{image: url({path}/widget files/checkbox_checked.png);}}'
+        StyleSheet += f'QCheckBox::indicator:unchecked {{image: url({path}/widget files/checkbox_unchecked.png);}}'
+        print(StyleSheet)
```

Comment:
These changes should probably be a separate PR, however I'm inclined to ignore that and leave it, it is at least a separate commit (though we tend to squash)...

It would not be hard to cherry pick it out and rebase this branch with out it, I can do it, if desired